ui/lib/config-ui/addon/components/messages/page/create-and-edit-message-form.hbs

```diff
@@ -9,21 +9,22 @@
 />
 
 <form id="message-create-edit-form" {{on "submit" (perform this.save)}} data-test-form="create-and-edit">
-  <div class="box is-sideless is-fullwidth is-marginless has-top-padding-s">
-    <Hds::Text::Body @tag="p" class="has-bottom-margin-l" data-test-form-subtext>
-      {{if @message.isNew "Create" "Edit"}}
-      a custom message for all users when they access a Vault system via the UI.
-    </Hds::Text::Body>
-
+  <div class="box is-sideless is-fullwidth is-marginless">
+    <NamespaceReminder @mode="create" @noun="message" />
     <MessageError @errorMessage={{this.errorBanner}} class="has-top-margin-s" />
 
     {{#each @message.formFields as |attr|}}
       <FormField @attr={{attr}} @model={{@message}} @modelValidations={{this.modelValidations}} class="has-bottom-margin-m">
-        <Messages::MessageExpirationDateForm @message={{@message}} @attr={{attr}} />
+        <Messages::MessageExpirationDateForm
+          @message={{@message}}
+          @attr={{attr}}
+          @onUpdateEndTime={{this.onUpdateEndTime}}
+          @isEndTimeValid={{or (not this.modelValidations) this.modelValidations.endTime.isValid}}
```

Comment:
To make this more extendable in the future, I'd handle this similarly to how validations are handled by `FormField` by passing in `@modelValidations` instead and then defining a getter using the `@attr` arg
```js
  get validationError() {
    const validations = this.args.modelValidations || {};
    const state = validations[this.args.attr.name];
    return state && !state.isValid ? state.errors.join(' ') : null;
  }
```

that way in the `<Messages::MessageExpirationDateForm>` template you can just check for `this.validationError`